.github/workflows/wheel_linux_x86_64_cuda.yml

```diff
@@ -78,7 +77,7 @@ jobs:
 
           CIBW_SKIP: "*-musllinux*"
 
-          CIBW_CONFIG_SETTINGS: --global-option=build_ext --global-option=--define="CMAKE_CXX_COMPILER=$(which g++);CMAKE_C_COMPILER=$(which gcc);LIGHTNING_RELEASE_TAG=master"
+          CIBW_CONFIG_SETTINGS: --global-option=build_ext --global-option=--define="LIGHTNING_RELEASE_TAG=master"
```

Comment:
Just for reference (nothing to do now), this may fail with future pip, and I currently don't see a better alternative: https://github.com/pypa/pip/issues/11859 Looks like this is also an unpopular move.